pkg/issue/issue.go

```diff
@@ -100,10 +100,11 @@ func ensure(ctx context.Context, c *github.Client, issues issues, owner, repo, p
 		} else {
 			footer = fmt.Sprintf("%v\n\n%v", oc.IssueFooter, operator.GitHubIssueFooter)
 		}
-		body := fmt.Sprintf("_This issue was automatically created by [Allstar](https://github.com/ossf/allstar/)._\n\n"+
-			"**Security Policy Violation**\n"+
+		ownerRepo := fmt.Sprintf("%s/%s", owner, repo)
+		body := fmt.Sprintf("_This issue was automatically created by [Allstar](https://github.com/ossf/allstar/) "+
+			"and refers to [%s](https://github.com/%s)._\n\n**Security Policy Violation**\n"+
```

Comment:
We only want to add this if `issueRepo != repo`, otherwise it will be confusing.